public/stylesheets/style.css

```diff
@@ -1,8 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-  padding: 50px;
-  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  /* border: 1px solid black; */
+  /* background-color: #f5f5f5; */
+  font-family: "Roboto", sans-serif;
+  margin: auto;
+  margin-top: 60px;
+  height: 500px;
+  width: 400px;
+}
+
+header {
+  background-color: #0d213a;
```

Comment:
Dont use elements as selectors when they are not scoped. Use either a class or ID or scope it